### PR TITLE
Mkhadiv/biped

### PR DESCRIPTION
**Pull request by majidkhadiv**
I added Bolt to the list of robots that we can generate motion for and created a jumping demo. I tested all the demos for solo8 and solo12 and they should be working.

### CODE REVIEW

File: momentumopt/python/momentumopt/robots/blmc_robot_wrapper.py [hunk 1]
```diff
@@ -256,6 +257,54 @@ def set_init_config(self):
...
+        self.q[7:] = q_dummy
+
+        # print(self.q)
+        self.set_configuration(self.q)
+
+
 ############################ For debugging ##########################################
 
-# robot = QuadrupedWrapper()
+# robot = BipedWrapper()
```
Comment (MaximilienNaveau) [inline momentumopt/python/momentumopt/robots/blmc_robot_wrapper.py:310]: If this is a comment here, should we remove it?

**Review by MaximilienNaveau — APPROVED** [review]
I have some minor comments to be addressed before merging.
But thanks a lot for the refactor!

**Review by jviereck — APPROVED** [review]
Looks good. Thanks for the changes.

File: momentumopt/python/momentumopt/robots/blmc_robot_wrapper.py (hunk 1 above)
Comment (majidkhadiv) [reply momentumopt/python/momentumopt/robots/blmc_robot_wrapper.py:310]: sure, any more comments?

Comment (MaximilienNaveau) [reply momentumopt/python/momentumopt/robots/blmc_robot_wrapper.py:310]: no, you are good to go

**Pull request merged**